graphql-java-support/src/test/java/com/apollographql/federation/graphqljava/LinkDirectiveProcessorTest.java

```diff
@@ -0,0 +1,86 @@
+package com.apollographql.federation.graphqljava;
+
+import com.apollographql.federation.graphqljava.exceptions.UnsupportedFederationVersionException;
+import java.util.Map;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class LinkDirectiveProcessorTest {
```

Comment:
this test class tests itself and not the `LinkDirectiveProcessor` so unsure whats the point